awx/main/models/workflow.py

```diff
@@ -438,9 +443,16 @@ def notification_templates(self):
                                               .filter(unifiedjobtemplate_notification_templates_for_started__in=[self]))
         success_notification_templates = list(base_notification_templates
                                               .filter(unifiedjobtemplate_notification_templates_for_success__in=[self]))
+        approval_notification_templates = list(base_notification_templates
+                                               .filter(workflowjobtemplate_notification_templates_for_approvals__in=[self]))
+        # Get Organization NotificationTemplates
+        if self.organization is not None:
```

Comment:
is fixing https://github.com/ansible/awx/issues/4712 just duplicating this for the other three notification types, and adding the appropriate notifications as above?

Reply:
@wenottingham Yes, as far as I can tell (doing that seemed to fix the issue).